src/Service/AuthManager.php

```diff
@@ -110,7 +124,13 @@ public function filterAccess($controllerName, $actionName)
                     $actionList=='*') {
                     if ($allow=='*')
                         return true; // Anyone is allowed to see the page.
-                    else if ($allow=='@' && $this->authService->hasIdentity()) {
+                    else if (($allow=='@' || is_array($allow)) && $this->authService->hasIdentity()) {
```

Comment:
I don't really like the @ behaviour. It cab be easily resolved with "user" role which cover every logged in user.